comms/Account.cpp

```diff
@@ -67,33 +118,18 @@ COMMS_API bool VerifyUser(IConfig * config, const CString &user, const CString &
     ESP_EXCEPTION_LOG3(response.Exceptions);
     if (server.VerifyUser(&request, &response) == SOAP_OK)
     {
-        SAFE_ASSIGN(retCode, response.retcode);
-        CString responseMessage;
-#if _COMMS_VER < 300800
```

Comment:
@GordonSmith I removed the ifdef because _ns1__UpdateUserResponse is the only response that has std::wstring *message;